gulpfile.js

```diff
@@ -13,9 +13,8 @@ require('laravel-elixir-vue-2');
  |
  */
 
-elixir((mix) => {
+elixir(function(mix) {
```

Comment:
Hi @punalsc - I was reading last night that the arrow function syntax here was introduced in Laravel 5.3. Could you revert these changes please?